### PR TITLE
ROC-4568 remove blue box of evidence listed from defendant view when submitted

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4568

### Change description
Remove blue box of evidence listed from defendant view when claimant submits none, in defendant full defense

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by singh-surjeet** [comment]
Approving but please make sure we get undefined is checked on evidence.rows

File: src/main/features/response/views/evidence.njk [hunk 1]
```diff
@@ -12,32 +12,31 @@
 {% block content %}
 <div class="grid-row">
   <div class="column-two-thirds">
-
-    <div class="description-block">
-      <h3 class="heading-small">Their evidence</h3>
+    {% if evidence.rows %}
```
Comment (andrewwa-kainos) [inline src/main/features/response/views/evidence.njk]: Can combine the ifs

Comment (robertparkinson) [reply src/main/features/response/views/evidence.njk]: if someone doesn't supply evidence is there an empty object or undefined?

Comment (ravinderpal) [reply src/main/features/response/views/evidence.njk]: Fixed!

**Pull request merged**